concepticondata/conceptlists.tsv

```diff
@@ -224,3 +224,14 @@ Marrison-1967-917	Marrison, Geoffrey Edward	1967		917	basic,areal	English	Naga
 Kolipakam-2018-100	Kolipakam, Vishnupriya and Jordan, Fiona M. and Dunn, Michael and Greenhill, Simon J. and Bouckaert, Remco and Gray, Russell D. and Verkerk, Annemarie	2018		100	basic,areal	English	Dravidian		Kolipakam2018		A 100 item wordlist for Dravidian languages based on [Kassian et al's list](:ref:Kassian-2010-116)		
 Reinhard-1970-337	Reinhard, Johan and Toba, Tim	1970		337	basic,areal	English	Kusunda		Reinhard1970		There are 337 items in the book.		
 Abraham-2018-307	Abraham, Binny and Sako, Kara and Kinny, Elina and Zeliang, Isapdaile	2018		307	basic,areal	English	Tshangla and Khobwa	https://www.sil.org/system/files/reapdata/12/94/06/129406896871137053545983118013436547675/silesr2018_009.pdf	Abraham2018	https://www.sil.org/system/files/reapdata/12/94/06/129406896871137053545983118013436547675/silesr2018_009.pdf	307 item wordlists for Tshangla and Khobwa dialects in Western Arunachal Pradesh.		
+Carling-2017-104a	Carling, Gerd (ed.)	2017	a	104		English		https://diacl.ht.lu.se/WordList/Index	Carling2017		Swadesh 100		
```

Comment:
Hi, several things here: author-format follows bibtex-standard, no editor sign. Use rather all official authors of the diacl paper in PLOS one. Then, pleas provide a description, saying, e.g., that this is what they thing is a Swadesh 100 list, and the description should end with a dot, as a normal sentence. Not just two words, ideally link it, like: 

```
According to the authors, this list represens the [100 concepts of Swadesh (1955)](:ref:Swadesh-1955-100). The four additional concepts in our Concepticon are due to ...
```
Maybe check the concepts to see why we have 104 for 100 ?

```
$ concepticon union Carling-2017-104a Swadesh-1955-100
```
The output will give you those concepts occurring only in one list. In fact, this is also some kind of a sanity check.

Reply:
Ah, more missing: please supply categories, as we do aleady in all other lists, meaning: areal list, basic list, etc. This is also straightforward, but they should not be missing.